modules/host/host.go

```diff
@@ -14,43 +14,197 @@ import (
+	// reasonably competitive.
+	defaultStoragePrice = modules.StoragePriceToConsensus(20e3) // 20 SC / GB / Month
 
 	// defaultCollateral defines the amount of money that the host puts up as
-	// collateral per-byte by default. Set to zero currently because neither of
-	// the negotiation protocols have logic to deal with non-zero collateral.
-	defaultCollateral = types.NewCurrency64(0)
+	// collateral per-byte by default. The collateral should be considered as
+	// an absolute instead of as a percentage, because low prices result in
+	// collaterals which may be significant by percentage, but insignificant
+	// overall. A default of 50 SC / GB / Month has been chosen, which is 2.5x
+	// the default price for storage. The host is expected to put up a
+	// significant amount of collateral as a commitment to faithfulness,
+	// because this guarantees that the incentives are aligned for the host to
+	// keep the data even if the price of siacoin fluctuates, the price of raw
+	// storage fluctuates, or the host realizes that there is unexpected
+	// opportunity cost in being a host.
+	defaultCollateral = types.NewCurrency64(50) // 50 SC / GB / Month
 
 	// defaultWindowSize is the size of the proof of storage window requested
 	// by the host. The host will not delete any obligations until the window
-	// has closed and buried under several confirmations.
+	// has closed and buried under several confirmations. For release builds,
+	// the default is set to 144 blocks, or about 1 day. This gives the host
+	// flexibility to experience downtime without losing file contracts. The
+	// optimal default, especially as the network matures, is probably closer
+	// to 36 blocks. An experienced or high powered host should not be
+	// frustrated by lost coins due to long periods of downtime.
 	defaultWindowSize = func() types.BlockHeight {
+		if build.Release == "dev" {
+			return 36 // 3.6 minutes.
+		}
+		if build.Release == "standard" {
+			return 144 // 1 day.
+		}
 		if build.Release == "testing" {
-			return 5
+			return 5 // 5 seconds.
+		}
+		panic("unrecognized release constant in host - defaultWindowSize")
+	}()
+
+	// maximumStorageFolderSize sets an upper bound on how large storage
+	// folders in the host are allowed to be. It makes sure that inputs and
+	// constructions are sane. While it's conceivable that someone could create
+	// a rig with a single logical storage folder greater than 128 TiB in size
+	// in production, it's probably not a great idea, especially when you are
+	// allowed to use many storage folders. All told, a single host on today's
+	// constants can support up to ~10 PB of storage.
+	maximumStorageFolderSize = func() uint64 {
+		if build.Release == "dev" {
+			return 1 << 33 // 8 GiB
 		}
 		if build.Release == "standard" {
-			return 144
+			return 1 << 47 // 128 TiB
+		}
+		if build.Release == "testing" {
+			return 1 << 20 // 1 MiB
 		}
+		panic("unrecognized release constant in host - maximum storage folder size")
```

Comment:
we could probably eliminate these checks with a simple `init` function in the build package:
```go
func init() {
    switch Release {
    case "dev", "standard", "testing":
    default:
        panic("unrecognized Release constant: " + Release)
    }
}
```